code/Data exploration.R

```diff
@@ -17,6 +17,7 @@ package_list <- c(
   , "lubridate"
   , "data.table"
   , "crayon"
+  ,"ggplot2"
```

Comment:
[no change required]: ggplot is actually part of tidyverse, so it gets loaded in the tidyverse package